tests/Functional/ContentConsultationRenderTest.php

```diff
@@ -0,0 +1,275 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme\Functional;
+
+use Behat\Mink\Element\NodeElement;
+use Drupal\Component\Utility\Html;
+use Drupal\oe_content_entity\Entity\CorporateEntityInterface;
+use Drupal\user\Entity\Role;
+use Drupal\user\RoleInterface;
+
+/**
+ * Test Consultation content type rendering.
+ */
+class ContentConsultationRenderTest extends ContentRenderTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'config',
+    'block',
+    'system',
+    'path',
+    'field_group',
+    'oe_theme_helper',
+    'oe_theme_content_consultation',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    // Give anonymous users permission to view entities.
+    Role::load(RoleInterface::ANONYMOUS_ID)
+      ->grantPermission('view published skos concept entities')
+      ->grantPermission('view published oe_contact')
+      ->save();
+  }
+
+  /**
+   * Tests Consultation full view mode rendering.
+   */
+  public function testTenderRendering(): void {
+    // Create document and image media.
+    $document = $this->createMediaDocument('consultation_document');
+    // Create general contact.
+    $contact = $this->createContactEntity('consultation_contact', 'oe_general', CorporateEntityInterface::PUBLISHED);
+
+    // Create a Consultation node.
+    $next_year = date('Y') + 1;
```

Comment:
Would be better use freezing time approach.
```
    $time = \Drupal::time();
    $time->freezeTime();
    $time->setTime($static_time->getTimestamp());
```
See ContentEventRenderTest::testEventRegistrationDateDescription(), ContentCallForProposalsRenderTest::testProposalRendering().